scene/resources/curve.h

```diff
@@ -263,6 +263,8 @@ class Curve3D : public Resource {
 	mutable Vector<size_t> points_in_cache;
 #endif
 
+	bool _closed = false;
```

Comment:
```suggestion
	bool closed = false;
```
This file doesn't apply `_` prefix to member variables, let's keep it uniform,

Reply:
Oh, okay, I see. I basically copied the PR for Line2D, but I guess I misunderstood how to use these, thank you 👍